jupyter/baseline/ubi9-python-3.12/setup-elyra.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -x
+
+# Set the elyra config on the right path
+# RHOAIENG-15626: Always copy our custom config to ensure it's up to date (install -D creates directory if needed)
+install -D -m 0644 /opt/app-root/bin/utils/jupyter_elyra_config.py /opt/app-root/src/.jupyter/jupyter_elyra_config.py
+chmod 2770 /opt/app-root/src/.jupyter/ 2>/dev/null || true  # Fix directory perms if created
+
+# create the elyra runtime directory if not present
+if [ ! -d $(jupyter --data-dir)/metadata/runtimes/ ]; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)